sections/kubectl_version.zsh

```diff
@@ -31,7 +31,7 @@ spaceship_kubectl_version() {
   [[ -z $kube_context ]] && return
 
   # if kubectl can't connect kubernetes cluster, kubernetes version section will be not shown
-  local kubectl_version=$(kubectl version --short 2>/dev/null | grep "Server Version" | sed 's/Server Version: \(.*\)/\1/')
+  local kubectl_version=$(kubectl version>/dev/null | grep "Server Version" | sed 's/Server Version: \(.*\)/\1/')
```

Comment:
Does this work?
`>/dev/null` seems to be hushing everything, not just errors.

Reply:
It does work, on the newer version >1.28 though. Didn't test on lower ones.

The --short args is not required and actually removed hence why it broke on newer version.